File: public/usage-examples/animations/create_animation-1-example.cpp
#include "splashkit.h"

int main()
{
    animation_script script = load_animation_script("WalkingScript", "kermit.txt");

    write_line("Creating animation...");
    animation anim = create_animation(script, "WalkFront");

    write_line("Animation name:");
    write_line(animation_name(anim));

    free_animation(anim);
    free_animation_script(script);

    return 0;
}

File: public/usage-examples/animations/create_animation-1-example.py
from splashkit import *

script = load_animation_script("WalkingScript", "kermit.txt")

print("Creating animation...")
anim = create_animation(script, "WalkFront")

print("Animation name:")
print(animation_name(anim))

free_animation(anim)
free_animation_script(script)
